### PR TITLE
Add password recovery and email confirmation

### PR DESCRIPTION
**Pull request by cvrkota**
We also added ability to destroy user from UI and we added username to login. 
Note: In config file we added a line 'config.allow_unconfirmed_access_for = 2.days', that allows us to login without confirmation as we are not receiving emails in development environment

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -27,10 +27,10 @@
   config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (spinecone) [inline config/initializers/devise.rb:27]: A problem for future developers to deal with 😄

**Comment by spinecone** [comment]
Looks good to me but I would love a review from one other person on the team!

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,6 +1,11 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
 
-    def home
-    end
-    
-end
```
Comment (niraulap) [inline app/controllers/application_controller.rb:6]: I am a little confused about what this end is doing

**Review by niraulap — APPROVED** [review]
Everything looks pretty good! I just had a tiny confusion about one of the ends in the ApplicationController. Other than that, I feel like I was able to follow-up with everything else and it looks good.

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (cvrkota) [reply app/controllers/application_controller.rb:6]: We deleted empty home, and added some protected methods! [Here](https://github.com/CSC322-Grinnell/anti-racism-library/pull/18/files#diff-55c5b7aecfb519d0e4880eaf2788eb6e) is how the whole file looks like

**Review by Coolshreshtha — APPROVED** [review]
seems to me the only changes are about the  additional usernames

**Pull request merged**